lib/node_modules/@stdlib/stats/base/dists/rademacher/pmf/benchmark/benchmark.js

```diff
@@ -0,0 +1,52 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2018 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var bench = require( '@stdlib/bench' );
+var randu = require( '@stdlib/random/base/randu' );
+var isnan = require( '@stdlib/math/base/assert/is-nan' );
+var pmf = require( '@stdlib/stats/base/dists/rademacher/pmf/lib' );
+var EPS = require( '@stdlib/constants/math/float64-eps' );
+var pkg = require( './../package.json' ).name;
+
+
+// MAIN //
+
+bench( pkg, function benchmark( b ) {
+	var p;
+	var x;
+	var i;
+
+	b.tic();
+	for ( i = 0; i < b.iterations; i++ ) {
+		x = ( randu()*( 2.0 + EPS ) ) - 1.0;
```

Comment:
I don't think you need the `EPS` adjustment here, if `[-1,1]` is inclusive.

```suggestion
 		x = ( randu()*2.0 ) - 1.0;
```